db/schema.rb

```diff
@@ -378,8 +382,10 @@
   end
 
   add_foreign_key "case_logs", "locations"
+  add_foreign_key "case_logs", "organisations", column: "organisations_id"
```

Comment:
Same, would also remove these here as there is no migration in this PR introducing these